redfish-core/lib/log_services.hpp

```diff
@@ -4435,83 +4474,50 @@ inline void getRedfishUriByDbusObjPath(
                         return;
                     }
 
-                    while (uriIdPos != std::string::npos)
+                    for (const auto& ancestorIface : ancestorsIfaces)
                     {
-                        std::string parentRedfishUri =
-                            redfishUri.substr(0, uriIdPos - 1);
-                        RedfishUriListType::const_iterator parentRedfishUriIt =
-                            std::find_if(
-                                redfishUriList.begin(), redfishUriList.end(),
-                                [parentRedfishUri](const auto& ele) {
-                                    return parentRedfishUri == ele.second;
-                                });
-
-                        if (parentRedfishUriIt == redfishUriList.end())
+                        bool foundAncestor{false};
```

Comment:
typically we init as  "bool foundAncestor = false;"

https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/50176/8/redfish-core/lib/ethernet.hpp

Reply:
Ok, I will change it as you mentioned